### ISSUE TITLE
Authenticate timesheet-Show employee level Data-authenticate data-DONOT PICK

### ISSUE THREAD

**Issue by ashokpkumar** [ashokpkumar/python_angular#128]
Scenario:
Once the employee submits time, the time data goes to the manager and is approved.
Once the time is approved, the status changes to approved in the backend.
Now the manager have to authenticate the time. for this we need to show the employee level data from backend .-This might have been already done in one of the endpoints
Now we need endpoints to authenticate the time submissions which will be taken during billing.

Frontend:
When i click on the approved times, in time submissions in manager mode, it should show tickmark for 

backend:
